src/frontend/web_application/src/scenes/ContactBook/presenter.jsx

```diff
@@ -62,17 +59,14 @@ class ContactBook extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      tags: [],
+      tags: [].concat(generateStateFromProps.bind(props)),
```

Comment:
ok let's include @themouette that introduced me this practice. It is a good to know pro/cons about that.

@ziir you declare using `generateStateFromProps` in `componentWillReceiveProps` is a mistake. Can you explain why ?